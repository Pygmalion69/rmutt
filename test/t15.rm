// scope qualifiers
r:
{{a="foo"} a {$b="bar"}} b " "
{{c:"foo"|"bar"} c {$d:"baz"|"quux"} d} d "\n";