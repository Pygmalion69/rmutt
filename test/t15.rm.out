foobar barquuxquux

foobar fooquuxquux

foobar barbazquux

foobar foobazquux

foobar barquuxbaz

foobar fooquuxbaz

foobar barbazbaz

foobar foobazbaz

